metpy/plots/declarative.py

```diff
@@ -928,6 +933,78 @@ def draw(self):
             self._need_redraw = False
 
 
+@exporter.export
+class SurfaceStationPlot(HasTraits):
+    """Plot surface station data from a Pandas Dataframe"""
+
+    parent = Instance(Panel)
+    _need_redraw = Bool(default_value=True)
+
+
+    field = List(Unicode())
+    field.__doc__ = """The variables that will be plotted on the stations
+
+
+    This is a list of variables. Typical variables include temperature, dewpoint,
+    mean_sea_level_pressure, skyc (skycover)
+    """
+
+
+    dattim = Instance(datetime, allow_none=True)
+    field.__dattim__ = """Datetime object that will easily allow user to subset data
```

Comment:
The parameter docs should be of the form `<variable name>.__doc__` for all. So in this case it should be `dattim.__doc__`